### PR TITLE
Type conversion

### PR DESCRIPTION
**Pull request by StefanEnberg**

### CODE REVIEW

File: src/main/java/qlik/jdbc/connector/ConnectorImpl.java [hunk 1]
```diff
@@ -4,92 +4,148 @@
 import io.grpc.*;
 import io.grpc.stub.StreamObserver;
 import qlik.connect.ConnectorGrpc;
+import qlik.connect.GrpcServer;
 import qlik.connect.GrpcServer.*;
```
Comment (wennmo) [inline src/main/java/qlik/jdbc/connector/ConnectorImpl.java]: Is this import redundant now?

File: example/postgres-image/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,5 @@
 FROM postgres:10.0
 COPY ./airports.csv /
```
Comment (wennmo) [inline example/postgres-image/Dockerfile]: Remove `airports.csv` now if new testdata is being used?

Comment (StefanEnberg) [reply example/postgres-image/Dockerfile]: I will keep both one the make sure that the correct types gets loaded and the other one to simulate a "bulk" test.

File: src/main/java/qlik/jdbc/connector/ConnectorImpl.java (hunk 1 above)
Comment (StefanEnberg) [reply src/main/java/qlik/jdbc/connector/ConnectorImpl.java]: We need both. One is for the GrpcServer class and the other one for all the subclasses under GrpcServer

File: example/docker-compose.yml [hunk 3]
```diff
@@ -12,12 +12,12 @@ services:
...
-#    image: qlikcore/engine:12.171.0
-#    ports:
-#      - 19076:9076
-#    command: -S AcceptEULA=yes -S EnableGrpcCustomConnectors=1 -S GrpcConnectorPlugins="jdbc,jdbc-connector:50051"
+  qix-engine:
+    container_name: qix-engine
+    image: qlikcore/engine:12.171.0
+    ports:
+      - 19076:9076
+    command: -S AcceptEULA=yes -S EnableGrpcCustomConnectors=1 -S GrpcConnectorPlugins="jdbc,jdbc-connector:50051"
```
Comment (peol) [inline example/docker-compose.yml]: Template this, we can't accept the EULA on end-users' behalf :)

**Pull request merged**